openshift-kube-apiserver/handlers/deprecatedapirequest/handler.go

```diff
@@ -0,0 +1,30 @@
+package deprecatedapirequest
+
+import (
+	"net/http"
+	"time"
+
+	apiclient "github.com/openshift/client-go/apiserver/clientset/versioned"
+	apiinformers "github.com/openshift/client-go/apiserver/informers/externalversions"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+	"k8s.io/client-go/rest"
+)
+
+// WithDeprecatedApiRequestHandler adds an http handler that logs requests to deprecated apis.
+func WithDeprecatedApiRequestHandler(handler http.Handler, config *genericapiserver.Config) http.Handler {
```

Comment:
this is actually not a new handler we add but a deprecated API usage counter mechanism, right?

In apiserver terms this construct is a filter, not a handler.

Reply:
Done. Renamed.